src/ActiveLogin.Authentication.BankId.AspNetCore/BankIdAuthenticationBuilderSchemeExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore
+{
+    public static class BankIdAuthenticationBuilderSchemeExtensions
+    {
+        internal static IBankIdAuthenticationBuilder AddScheme(this IBankIdAuthenticationBuilder builder, string authenticationScheme, string displayName, Action<BankIdAuthenticationOptions> configureOptions, Action<BankIdAuthenticationOptions> defaultConfigureOptions)
+        {
+            builder.AuthenticationBuilder.Services.Configure(authenticationScheme, defaultConfigureOptions);
+
+            builder.AuthenticationBuilder.AddScheme<BankIdAuthenticationOptions, BankIdAuthenticationHandler>(
+                authenticationScheme,
+                displayName,
+                configureOptions
+            );
+
+            return builder;
+        }
+
+
+        public static IBankIdAuthenticationBuilder AddCustom(this IBankIdAuthenticationBuilder builder, Action<BankIdAuthenticationOptions> configureOptions)
+            => AddCustom(builder, BankIdAuthenticationDefaults.CustomAuthenticationScheme, BankIdAuthenticationDefaults.CustomDisplayName, configureOptions);
+
+        public static IBankIdAuthenticationBuilder AddCustom(this IBankIdAuthenticationBuilder builder, string authenticationScheme, Action<BankIdAuthenticationOptions> configureOptions)
+            => AddCustom(builder, authenticationScheme, BankIdAuthenticationDefaults.CustomDisplayName, configureOptions);
+
+        public static IBankIdAuthenticationBuilder AddCustom(this IBankIdAuthenticationBuilder builder, string authenticationScheme, string displayName, Action<BankIdAuthenticationOptions> configureOptions)
+            => AddScheme(builder, authenticationScheme, displayName, configureOptions, options =>
+            {
+                options.CallbackPath = BankIdAuthenticationDefaults.CustomCallpackPath;
+            });
+
+
+        public static IBankIdAuthenticationBuilder AddSameDevice(this IBankIdAuthenticationBuilder builder, Action<BankIdAuthenticationOptions> configureOptions)
```

Comment:
Could we may be add one more overload method for BankId that does not take any arguments, or make configuredOptions equal null for the selected one. Then we can simplify the template code to:
```
.AddBankId(builder =>
                {
                    builder
                        .UseDevelopmentEnvironment()
                        .AddSameDevice()
                        .AddOtherDevice();
                });
```